recipes/funchook/all/test_package/conanfile.py

```diff
@@ -0,0 +1,32 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
```

Comment:
```suggestion
```

Do not force the package ID on test package

Reply:
Can you explain how configure() in the test package affects the package ID?  Or point me at the right documentation?  Sorry, I'm not really following this comment.

Basically, I thought the idea behind deleting these settings was to simply mark this package as only requiring C, not C++.  But I'm kinda a noob at this....